.circleci/config.yml

```diff
@@ -7,7 +7,7 @@ executors:
   # also be updated.
   golang:
     docker:
-      - image: cimg/go:1.23
+      - image: cimg/go:1.24
```

Comment:
This should be 1.25.

I think you misunderstand how this works.
* This configures the current Go version for builds.
* The `go.mod` file simply specifies the _minimum_ Go version.